x-pack/legacy/plugins/siem/public/pages/home/index.tsx

```diff
@@ -92,46 +92,36 @@ export const HomePage: React.FC = () => (
 
                 <Switch>
                   <Redirect exact from="/" to={`/${SiemPageName.overview}`} />
-                  <Route
-                    path={`/:pageName(${SiemPageName.overview})`}
-                    render={() => <Overview />}
-                  />
+                  <Route path={`/:pageName(${SiemPageName.overview})`}>
+                    <Overview />
+                  </Route>
                   <Route
                     path={`/:pageName(${SiemPageName.hosts})`}
-                    render={({ location, match }) => (
-                      <HostsContainer location={location} url={match.url} />
-                    )}
+                    render={({ match }) => <HostsContainer url={match.url} />}
```

Comment:
I am wondering here since we have the new version of react-router, we might be able to use their hook to get the value inside of the component without the props.